WzComparerR2/CharaSimControl/ItemTooltipRender2.cs

```diff
@@ -192,16 +120,63 @@ public override Bitmap Render()
                             if (imgNode != null)
                             {
                                 Item item = Item.CreateFromNode(imgNode, PluginManager.FindWz);
+                                if (removeTimeLimited)
+                                {
+                                    item.Props[ItemPropType.timeLimited] = 0;
+                                }
                                 if (item != null)
                                 {
-                                    recipeItemBmp = RenderLinkRecipeItem(item);
+                                    recipeItemBmps.Add(RenderLinkRecipeItem(item));
                                 }
 
                                 break;
                             }
                         }
                     }
                 }
+            };
+
+            //图纸相关
+            int recipeID;
+            if (this.item.Specs.TryGetValue(ItemSpecType.recipe, out recipeID))
+            {
+                int recipeSkillID = recipeID/10000;
+                Recipe recipe = null;
+                //寻找配方
+                Wz_Node recipeNode = PluginBase.PluginManager.FindWz(string.Format(@"Skill\Recipe_{0}.img\{1}", recipeSkillID, recipeID));
+                if (recipeNode != null)
+                {
+                    recipe = Recipe.CreateFromNode(recipeNode);
+                }
+                //生成配方图像
+                if (recipe != null)
+                {
+                    if (this.LinkRecipeInfo)
+                    {
+                        recipeInfoBmp = RenderLinkRecipeInfo(recipe);
+                    }
+
+                    if (this.LinkRecipeItem)
+                    {
+                        int itemID = recipe.MainTargetItemID;
+                        AppendGearOrItem(itemID, true);
+                    }
+                }
+            }
+
+            int value;
+            if (this.item.Props.TryGetValue(ItemPropType.dressUpgrade, out value))
+            {
+                int itemID = value;
+                AppendGearOrItem(itemID, false);
+            }
+
+            if (this.item.AddTooltips.Count > 0)
+            {
+                foreach (int itemID in item.AddTooltips)
+                {
+                    AppendGearOrItem(itemID, false);
```

Comment:
```suggestion
                    AppendGearOrItem(itemID, true);
```

보니까 `addTooltip`에 대해서도 timeLimited를 없애줘야겠습니다.

![image](https://user-images.githubusercontent.com/6624567/92326014-fcc24f80-f089-11ea-94e4-366be34971d6.png)